action/protocol/staking/protocol.go

```diff
@@ -376,10 +376,18 @@ func (p *Protocol) ReadState(ctx context.Context, sr protocol.StateReader, metho
 		err = errors.New("corresponding method isn't found")
 	}
 	if err != nil {
-		return nil, err
+		return nil, uint64(0), err
+	}
+	bytes, err := proto.Marshal(resp)
```

Comment:
nit: `bytes` -> `data`. (`bytes` is a little bit similar to go type `byte`)

Reply:
fixed in latest commit

Reply:
addressed on latest commit